checkov/main.py

```diff
@@ -331,6 +335,14 @@ def run(banner: str = checkov_banner, argv: List[str] = sys.argv[1:]) -> Optiona
     git_configuration_folders = [os.getenv("CKV_GITHUB_CONF_DIR_PATH", default_github_dir_path),
                                  os.getcwd() + '/' + os.getenv('CKV_GITLAB_CONF_DIR_NAME', 'gitlab_conf')]
 
+    def commit_repository() -> Optional[str]:
```

Comment:
you could also change it to a normal function and pass `config` as a parameter to it